zerver/lib/actions.py

```diff
@@ -3257,6 +3267,23 @@ def name(sub):
                 state['twenty_four_hour_time'] = event["setting"]
             if event['setting_name'] == 'left_side_userlist':
                 state['left_side_userlist'] = event["setting"]
+        elif event['type'] == "update_global_notifications":
+            if event['notification_name'] == "enable_stream_desktop_notifications":
```

Comment:
So this part seems like it has a common pattern for the if statements:

    if event['notification_name'] == something:
         state[something] = event['settings']

Any way you could simplify those and avoid all the `if`s and `elif`s?